client/main.ml

```diff
@@ -86,12 +86,16 @@ let list_refs repo =
            Fmt.pr "%s %s (%a)@." hash gref Client.Build_status.pp status)
          refs
 
+let to_ts_string (tt : float) =
+  let ts = Timedesc.of_timestamp_float_s tt in
+  Timedesc.to_iso8601 @@ Option.get ts
```

Comment:
Can't it be more secure to use `Option.value ~default:"not available"` instead of `get`?

Reply:
Sure. I'll use `-` as a default, same as below.